samples/seat_massager/provider/src/provider_impl.rs

```diff
@@ -92,6 +94,8 @@ impl ProviderImpl {
 
 #[tonic::async_trait]
 impl DigitalTwinProvider for ProviderImpl {
+    type StreamStream = Pin<Box<dyn Stream<Item = Result<StreamResponse, Status>> + Send>>;
```

Comment:
This name is generated by a gRPC convention.  The first term reflects the operation name. The second term must be "Stream".

Reply:
It's still an unusual type name. can we consider changing the type name, or perhaps changing the API name to something other than "stream" so this oddity doesn't show up?

Reply:
What name other operation name than "Stream" do you suggest?  Anything that I use must have "stream" somewhere in the name.  I could call it "OpenStream", but then we would need to define and use a type called "OpenStreamStream".  I thought that "StreamStream" was the best of a list of bad options.

Reply:
yeah this might be the best option, let's just leave it. somewhat annoying that grpc is forcing this naming on us, but this is conceptually the stream for the stream operation